changelog/13348.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
I think this should be `improvement` not `feature`, as `feature` usually refers to large changes in a major release.